USBHelperLauncher/Program.cs

```diff
@@ -195,6 +196,40 @@ static void Main(string[] args)
                 }
             }
 
+            // Make sure that FiddlerCore's key container can be accessed
+            string keyContainerName = FiddlerApplication.Prefs.GetStringPref("fiddler.certmaker.bc.KeyContainerName", "FiddlerBCKey");
+            try
+            {
+                CspParameters cspParams = new CspParameters();
```

Comment:
Can be simplified:
```
CspParameters cspParams = new CspParameters { KeyContainerName = keyContainerName };
```
I should get around to doing this to the rest of the codebase soon.

Reply:
Should all be fixed now.